pkg/instance/instance.go

```diff
@@ -230,21 +230,16 @@ func (i *Instance) SetInstanceType(instanceType InstanceType) {
 }
 
 // SetImage sets the image of the instance.
-// When calling in state 'Started', make sure to call AddVolume() before.
-// It is only allowed in the 'None' and 'Started' states.
+// It is not allowed in the 'Committed' and 'Started' states.
 func (i *Instance) SetImage(ctx context.Context, image string) error {
-	if !i.IsInState(StateNone, StateStarted) {
```

Comment:
There are two use cases for `SetImage`
1. To get from `StateNone` to `StatePreparing` after a new instance was created
2. To restart an Instance with a new image after the Instance already started.

Reply:
Removing the `StateNone` here means we can't get from `StateNone` to `StatePreparing` by setting an existing image.